.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+
+<!-- Check the [Issues](https://github.com/octokit/rest.js/issues) on Github to ensure one doesn't already exist. -->
```

Comment:
No need to use markdown here. And maybe combine all comments into one? Maybe something like this

```
<!--
  1. Please make sure there is no open issue yet: https://github.com/octokit/rest.js/issues?utf8=%E2%9C%93&q=is%3Aissue
  2. If this is a bug report, please create a reproducible test case on runkit.com. Example: https://runkit.com/gr2m/5aa034f1440b420012a6eebf
  3. If this is a feature request, please share the motivation for the new feature and how you would implement it
--> 
```